internal/util/password/password.go

```diff
@@ -14,3 +14,20 @@
 
 // Package password provides utilities for password hashing and verification.
 package password
+
+// Password wraps a password string.
+//
+// It exist mainly to avoid issues when multiple string parameters are used.
```

Comment:
Another reason is to try to hide values of that type from logs in the future.

That's experimental. I will either update more code to use that type or remove it in future PRs.